e2e/tests-dfx/network.bash

```diff
@@ -81,6 +81,6 @@ teardown() {
 
     setup_actuallylocal_shared_network
 
-    assert_command_fail dfx build --network actuallylocal
+    assert_command_fail dfx build e2e_project_frontend --network actuallylocal
```

Comment:
Not necessarily required, but I ran into problems while debugging with this not being explicit. The error message below refers explicitly to `e2e_project_frontend`, so if `[...]_backend` is built first, then the error message doesn't match even though nothing really went wrong